### PR TITLE
feat(j-s): Appeals court - Add a button for prosecutors to appeal case to COA

### PR DESCRIPTION
**Pull request by oddsson**
# Appeals court - Add a button for prosecutors to appeal case to COA

[Asana](https://app.asana.com/0/1199153462262248/1204127104511325/f)

## What

- Add a button for prosecutors to appeal case to appeals court
- Remove current "appeal to appeals court" functionality. 

## Why

This is step one in creating a "Appeal to court of appeals" flow.

## Screenshots / Gifs

Screenshot 2023-03-20 at 13 23 39

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx [hunk 1]
```diff
@@ -487,500 +494,531 @@ export const SignedVerdictOverview: React.FC = () => {
...
-                <Text as="h1" variant="h1">
-                  {titleForCase(formatMessage, workingCase)}
-                </Text>
-              </Box>
+    <>
+      {workingCase.courtEndTime &&
+        !workingCase.isAppealDeadlineExpired &&
+        user?.role &&
+        isProsecutionRole(user.role) &&
+        features.includes(Feature.APPEAL_TO_COURT_OF_APPEALS) && (
```
Comment (oddsson) [inline apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx:502]: Am I thinking about this correctly? I tend to get this feature hiding logic wrong 🙈

**Review by brynjarorng — APPROVED** [review]
LGTM(Devops)

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx [hunk 2]
```diff
@@ -82,6 +85,9 @@ import * as constants from '@island.is/judicial-system/consts'
 import AppealSection from './Components/AppealSection/AppealSection'
 import { CourtRecordSignatureConfirmationQuery } from './courtRecordSignatureConfirmationGql'
 import ModifyDatesModal from './Components/ModifyDatesModal/ModifyDatesModal'
+import { signedVerdictOverview as strings } from './SignedVerdictOverview.strings'
+import { getAppealEndDate } from '@island.is/judicial-system-web/src/utils/stepHelper'
+import { FeatureContext } from '@island.is/judicial-system-web/src/components/FeatureProvider/FeatureProvider'
```
Comment (unakb) [inline apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx:90]: Please move imports up 😊

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx (hunk 1 above)
Comment (unakb) [reply apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx:502]: The way I understand the feature hiding, this is correct 😄

**Comment by oddsson** [comment]
Closed in favour of https://github.com/island-is/island.is/pull/10582

**Pull request closed without merging**